bin/BuildPackages.sh

```diff
@@ -112,7 +112,7 @@ notice \
 "you require!"
 
 echo_run() {
-  echo "$@"
+  notice "Running $(printf "'%s' " "$@")"
```

Comment:
No, that simply notice line would not be enough. Note the printf is called with a format string that only caputes a single argument. In that situation, printf applies the format string to each argument in sequence. It thus prints out each argument quoted with single quotes. That means you can tell the difference between  `./configure CFLAGS=-m32 LDFLAGS=-m32` (which gets printed as  `'./configure' 'CFLAGS=-m32' 'LDFLAGS=-m32'` and the incorrect ` `./configure "CFLAGS=-m32 LDFLAGS=-m32"` (which gets printed as  `'./configure' 'CFLAGS=-m32 LDFLAGS=-m33'`)

As to why the code runs correct: Because those are the (admitedly bizarre) rules of POSIX shell quoting...